PyInstaller/building/templates.py

```diff
@@ -103,3 +105,11 @@
              icon=%(icon)s,
              bundle_identifier=%(bundle_identifier)s)
 """
+
+splashtmpl = """splash = Splash('%(splash_image)r',
```

Comment:
Whoops - didn't spot this because I built my spec file manually. If we're using `repr`, we need to remove the quotes as well, because `repr` includes those.
```suggestion
splashtmpl = """splash = Splash(%(splash_image)r,
```

Reply:
It builds now, but doesn't show the splash image.

Can you send/link me a working spec file?

Reply:
Hmm... well, I'll have to fix that... Thanks for noticing!

A working spec file can be found at `tests\functional\specs\spec_with_splash.spec`, you might want to have a look at it. This spec is executed by pytest. If you want, you can run the test yourself and see if it works. The command for this would be `pytest tests\functional -k test_pyi_splash`.

Since this is still a PR, the bootloader has not yet been recompiled, so you will need to do this if you haven't already.